.github/workflows/update-owasp-db.yaml

```diff
@@ -0,0 +1,20 @@
+name: Update Vulnerability Database
+
+on:
+  workflow_dispatch:
```

Comment:
committed on this PR so that it can be run & tested later on in a different PR using the trigger